server/TossupRoom.js

```diff
@@ -290,6 +306,23 @@ class TossupRoom {
             });
             this.adjustQuery(['setName', 'packetNumbers'], [message.value, message.packetNumbers]);
             break;
+        
+        case 'toggle-bot':
+            if (message.bot == true) {
+                this.quizBot = new QuizBot();
+                this.createPlayer('QuizBotId');
+
+                this.sendSocketMessage({
+                    type: 'join',
+                    isNew: true,
+                    userId: 'QuizBotId',
+                    username: '🤖QuizBot🤖',
```

Comment:
This username should be blacklisted from account creation or names in public games, as that could lead to impersonation/abuse.